rts/lra/test/pom.xml

```diff
@@ -137,8 +287,8 @@
                                         <argument>${lra.coordinator.log.file.module.narayana}</argument>
                                         <argument>-DRecoveryEnvironmentBean.periodicRecoveryPeriod=10</argument>
                                         <argument>-DRecoveryEnvironmentBean.recoveryBackoffPeriod=5</argument>
-                                        <argument>-DObjectStoreEnvironmentBean.objectStoreDir=${project.build.directory}/lra_txn_logs</argument>
-                                        <argument>-DObjectStoreEnvironmentBean.communicationStore.objectStoreDir=${project.build.directory}/lra_txn_logs</argument>
+                                        <argument>-DObjectStoreEnvironmentBean.objectStoreDir=${project.build.directory}/quarkus_lra_objectstore</argument>
```

Comment:
`lra_txn_logs` is more generic. Did you make this change so that a module can be tested against multiple containers concurrently. If they only ever run against one container in a single execution then we should use a generic name since it slows down developers and inhibits scripting. Besides the contents of filesystem based stores should be the same regardless of which container we run against.

Reply:
+1

Reply:
To each of the four points:

1. So you are saying that the build breaks if you use a generic name (such as lra-txn-logs).
2. I am talking about developers wanting to write their own scripts to manager their own work. Plus the lack of consistency incurs greater mental effort and when our time has multiple demands upon it just adds overhead and makes working with the code base a less pleasant experience.
3. I discussed and addressed this point in my review comment.
4. Yes make the name consistent (lra-logs or whatever) regardless of which container is in use (unless doing so breaks the build as per my response to point 1).

Reply:
Ok, I changed this back to `lra_txn_logs`.

Reply:
I'm resolving this discussion. If I missed something to be done, please, reopen it.